### PR TITLE
Path: Add marlin_post.py

### PR DESCRIPTION
**Pull request by glhasson**
Modified grbl_post.py into marlin_post.py.
Tested marlin_post.py extensively by milling actual parts
on a three axis milling machine running Marlin 2.0.7.bugfix

### CODE REVIEW

**Comment by sliptonic** [comment]
Looks like your first PR.  Welcome!  A couple of notes:

If you haven't done so, it's usually a good idea to post on the forum and start a discussion about your change.  It isn't mandatory but friendly.

Since this is a new file, it's difficult to look at it and see what's different from grbl_post.   Can you summarize how this post differs?

Also, you don't need to merge master back into your branch and doing so makes the history messy.  Especially with a brand new file that can't have a merge conflict, you don't need to keep the PR up to date with the master branch.  

With a more complex PR, If you want to keep it up to date, the way to do it is to rebase your local branch on upstream/master then push/force your local branch to github.  The PR will automatically update.

**Comment by glhasson** [comment]
Thank you sliptonic, for the guidance!
I am in the process of providing better documentation. It should be ready tomorrow.
Is it possible to contact you via email so I can ask a few questions?

**Comment by sliptonic** [comment]
Best way to reach me is on the gitter channel.  A lot of that work on Path are there.  You can log in with your github account.  https://gitter.im/FreeCAD/Path
From there you can also ping me for a private conversation if you want.

**Comment by glhasson** [comment]
Thanks sliptonic, I'll try that tomorrow. I live in Spain and it is a bit late to start something now.
